src/modules/updateDom.js

```diff
@@ -1,13 +1,12 @@
-import storage from './storage';
-
 const scoresContainer = document.getElementById('scores-container');
 
-export default function update() {
+export default function update(scores) {
```

Comment:
- [ ] it would be better if you use the `Arrow function` instead of `function keyword` in order to practice what you've learned and also you will be familiar with both